Fix issues with updating channel

drillprop opened the pull request:
Fix #3224 
While working on this task I found that updating a channel is pretty buggy right now. When testing, make sure that you manage: 
- create a new channel and set an avatar and cover
- update an existing channel without an avatar and cover
- update an existing channel with an avatar and cover
- update only one asset on an existing channel
- switch between channels while the edit channel form is open(check if the content in the form is updated correctly)
- update only one asset on an existing channel - either avatar or cover, and once this asset is uploaded and processed, try to update the channel again.

Just FYI, deleting assets won't be covered in this PR. It's still not working fully and we have a separate issue for this.

drillprop committed:
fix issues with resetting and not showing avatars

drillprop committed:
fix issues with not setting cover correctly

drillprop committed:
fix issue with updating cover once avatar is uploaded

drillprop committed:
remove isDirty from reset effect

drillprop committed:
fix issue with updating cover once avatar is uploaded(hopefully)

rafalpawlow commented on packages/atlas/src/providers/assets/assets.hooks.ts:
```diff
@@ -29,7 +29,9 @@ export const useAsset = (dataObject?: StorageDataObjectFieldsFragment | null) =>
 export const useRawAsset = (contentId: string | null) => {
-  return useAssetStore((state) => (contentId ? state.assets[contentId] : null))
+  return useAssetStore((state) => {
+    return contentId ? state.assets[contentId] : null
```
Why did you make this change?

rafalpawlow reviewed (changes requested):
1. All scenarios work pretty solid however, I have one doubt. After I upload an avatar I cannot update another one, first I need to delete the previously updated one. Not sure if this is expected behavior. Check the video below: 

https://user-images.githubusercontent.com/13706246/195081777-db276fe1-4e3d-46cb-b6d2-9a1fbfaf1100.mov

2. Do you think we could try to get rid of this as part of this PR? It appears after the channel is created and looks worrying
![Zrzut ekranu 2022-10-11 o 13 34 22](https://user-images.githubusercontent.com/13706246/195081954-d361d3ca-7436-4805-8e9b-651cb023f673.png)

rafalpawlow commented:
After a discussion with @drillprop we agreed that 1st point is expected behavior.

drillprop committed:
cr fixes

drillprop commented:
> 2. Do you think we could try to get rid of this as part of this PR? It appears after the channel is created and looks worrying
>    ![Zrzut ekranu 2022-10-11 o 13 34 22](https://user-images.githubusercontent.com/13706246/195081954-d361d3ca-7436-4805-8e9b-651cb023f673.png)

Fixed, I don't think we should worry about this warning.  The `NewAccount` event
is marked as a system event for some reason, I updated the if statement in `parseExtrinsicEvents`
